contributing.md

```diff
@@ -46,8 +46,18 @@ you'll need a recent nightly compiler to build it.
 git clone https://github.com/rust-lang/rls.git
 cd rls
 cargo build --release
-```
```

Comment:
I think that's deleted by mistake

Reply:
Now it's 6 backticks instead of none 😅 
No worries, I'll fix that myself - thanks for writing that up!